src/renderer/features/network/NetworkSelector/model/network-selector-model.ts

```diff
@@ -1,18 +1,20 @@
-import { createEvent, sample, attach } from 'effector';
-import { delay, spread } from 'patronum';
+import { createEvent, sample, createEffect, createStore } from 'effector';
+import { spread } from 'patronum';
 
 import { networkModel, networkUtils } from '@entities/network';
-import { ChainId, ConnectionType, RpcNode } from '@shared/core';
+import { ChainId, ConnectionType, RpcNode, Connection } from '@shared/core';
+import { storageService } from '@shared/api/storage';
 
 const lightClientSelected = createEvent<ChainId>();
 const autoBalanceSelected = createEvent<ChainId>();
 const rpcNodeSelected = createEvent<{ chainId: ChainId; node: RpcNode }>();
-const chainDisabled = createEvent<ChainId>();
+const networkDisabled = createEvent<ChainId>();
 
-const updateConnectionFx = attach({ effect: networkModel.effects.updateConnectionFx });
-const disconnectProviderFx = attach({ effect: networkModel.effects.disconnectProviderFx });
+const $reconnectMap = createStore<Record<ChainId, boolean>>({});
```

Comment:
Map with chains that requires reconnect when changing from one active node to another